modules/RTC/TPCUtils.js

```diff
@@ -379,8 +383,15 @@ export class TPCUtils {
                         transceiver.direction = TransceiverDirection.SENDRECV;
                     }
 
-                    // Add the new track to the list of local tracks.
-                    this.pc.localTracks.set(newTrack.rtcId, newTrack);
+                    // Avoid configuring the encodings on chromium since the encoding params are read-only
+                    // until the renegotation is done after the track is replaced on the sender.
+                    const promise = browser.isChromiumBased() ? Promise.resolve() : this.setEncodings(newTrack);
```

Comment:
isn't that the case for any browser? what does the spec say? I thought the encodings are set via "transceiver init" or something like that

Reply:
No, it doesn't seem to be the case on all the browsers. When the track is added initially, encodings are specified in transceiverInit for Chrome/Safari but FF ignores config transceiverInit so the encodings are configured after addTrack. This case is for when a track is removed the pc and then added back to the pc using replaceTrack. This is the case where chrome makes them read-only until the reneg happens.